src/encoded/static/components/item-pages/ExperimentSetView.js

```diff
@@ -416,47 +420,25 @@ export class ProcessedFilesStackedTableSection extends React.PureComponent {
      * Else, if any bigwig files exist, return those.
      * Else, return null.
      */
-    static findAllFilesToVisualize(context){
+    static findAllFilesToVisualize = memoize(function(context){
         return OtherProcessedFilesStackedTableSectionPart.findAllFilesToVisualize( // <- this function might be moved closer to HiGlassContainer in near future.
             expFxn.allProcessedFilesFromExperimentSet(context)
         );
-    }
-
-    constructor(props){
-        super(props);
-        this.state = {
-            'currentlyVisualizedFiles' : ProcessedFilesStackedTableSection.findAllFilesToVisualize(props.context), // TODO: May change, act on some 'currentVisualizedFileType' param or state, etc.
-            'filesWithMetrics' : ProcessedFilesQCStackedTable.filterFiles(props.files)
-        };
-        this.columnHeaders = ProcessedFilesStackedTableSection.extendedColumnHeaders();
-    }
-
-    componentWillReceiveProps(nextProps){
```

Comment:
In this file we factor out deprecated `componentWillReceiveProps` and replace it with memoized static functions, similar to what we did in file-tables.js.

Functionally everything should stay same.